virttest/utils_test/libvirt.py

```diff
@@ -1438,47 +1438,55 @@ def create_hostdev_xml(pci_id, boot_order=None, xmlfile=True,
     return hostdev_xml.xml
 
 
-def create_controller_xml(contr_dict, oper="get_xml", vm_name=None):
+def add_controller(vm_name, contr):
```

Comment:
I assume we don't need return controller back  since you have newly added contr as input parameter

Reply:
This is originally copied from existing codes.
Update..